docs/primer/releases_and_records.md

```diff
@@ -3,7 +3,7 @@
 ```{admonition} Objectives
 :class: note
 
-Open Contracting Data Standard (OCDS) data is published and updated over time using two JSON document types: “releases” and “records”.  This page will:
+Open Contracting Data Standard (OCDS) data is published and updated over time using two types of JSON text: “releases” and “records.” This page will:
```

Comment:
```suggestion
Open Contracting Data Standard (OCDS) data is published and updated over time using two types of JSON text: “releases” and “records”. This page will:
```

Reply:
That's British style punctuation. We use American style punctuation and spelling.